### ISSUE TITLE
NVDA 2024.3.1 is not registered in the transform json, causing ad-on store to report failure when retrieiving add-on data

### ISSUE THREAD

**Issue by josephsl** [nvaccess/addon-datastore-transform#27]
Hi,

This might be due to the logic in the Git tag to transform data transfer process:

Suppose the API transform data is defined thus:

* current
* next (current+1) beta
* development (alpha)

Suppose a new tag is created called current.1, ideally placed between current and next beta 1 (in the json, beta doesn't matter). However, when a Git tag is sent to transform data, this fact is not recorded.

Real exmaple: NVDA 2024.3 (current) and 2024.4 (next) are presetn but not 2024.3.1 (current.1). This causes add-on store to report failure when retrieving add-on data information because 2024.3.1 doesn't exist in API json file.

Thanks.

**Comment by seanbudd** [comment]
thanks for opening this issue. It seems like we need to update documentation for patch releases to ensure this is covered
